conf/base.config

```diff
@@ -11,43 +11,22 @@
 
 process {
 
-  container = params.container
-
-  cpus = { check_max( 8 * task.attempt, 'cpus' ) }
-  memory = { check_max( 16.GB * task.attempt, 'memory' ) }
+  cpus = { check_max( 1 * task.attempt, 'cpus' ) }
+  memory = { check_max( 8.GB * task.attempt, 'memory' ) }
```

Comment:
All processes in the pipeline will run with these parameters?

Reply:
for now I just left it like it is, maybe we can adapt this in the future